File: app/src/main/java/br/com/ribamar/concrete/model/pojos/repositories/Repositories.java

package br.com.ribamar.concrete.model.pojos.repositories;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Repositories {

    @SerializedName("incomplete_results")
    private Boolean incompleteResults;
    @Expose
    private List<Item> items;
    @SerializedName("total_count")
    private Long totalCount;

    public Boolean getIncompleteResults() {
        return incompleteResults;
    }

    public void setIncompleteResults(Boolean incompleteResults) {
        this.incompleteResults = incompleteResults;
    }

    public List<Item> getItems() {
        return items;
    }

    public void setItems(List<Item> items) {
        this.items = items;
    }

    public Long getTotalCount() {
        return totalCount;
    }

    public void setTotalCount(Long totalCount) {
        this.totalCount = totalCount;
    }

}

File: app/src/main/java/com/example/desafio_android/pojo/Repositories.kt
package com.example.desafio_android.pojo

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName


class Repositories {
    @SerializedName("total_count")
    @Expose
    var totalCount: Int? = null
    @SerializedName("incomplete_results")
    @Expose
    var incompleteResults: Boolean? = null
    @SerializedName("items")
    @Expose
    var items: List<Item>? = null

}